modules/monitoring-modifying-retention-time-for-prometheus-metrics-data.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// monitoring/configuring-the-monitoring-stack.adoc
+
+[id='modifying-retention-time-for-prometheus-metrics-data-{context}']
+= Modifying retention time for Prometheus metrics data
+
+By default, the Prometheus Cluster Monitoring stack configures the retention time for Prometheus data to be 15 days. You can modify the retention time to change how soon the data is deleted.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Put your retention time configuration under `data/config.yaml`:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: cluster-monitoring-config
+    namespace: openshift-monitoring
+  data:
+    config.yaml: |+
+      prometheusK8s:
+        retentionTime: *_time specification_*
+----
++
+Substitute `<time specification>` with a number directly followed by `ms` (milliseconds), `s` (seconds), `m` (minutes), `h` (hours), `d` (days), `w` (weeks), or `y` (years).
++
+For example, to configure retention time to be 24 hours, use:
++
+[source,yaml,subs=quotes]
+----
+data:
+ config.yaml: |
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: cluster-monitoring-config
+    namespace: openshift-monitoring
+  data:
+    config.yaml: |+
+      prometheusK8s:
+        retentionTime: *24h*
+----
+
+. Apply the new configuration:
```

Comment:
No need for this step, and the command will return error.
Just save the configmap is OK, the configuration will be applied.

#oc -n openshift-monitoring apply configmap cluster-monitoring-config
Error: required flag(s) "filename" not set